phaseshifts/lib/_fortran_lib.py

```
@@ -79,6 +79,22 @@ def is_module_importable(module):  # (str) -> bool
     return is_importable
 
 
+def print_package_tree(root_dir=None, prefix=""):
```

Ruff (0.14.10)

Missing return type annotation for private function `print_package_tree`

Add return type annotation: `None`

(ANN202)